yellowbrick/cluster/silhouette.py

```diff
@@ -194,8 +195,15 @@ def finalize(self):
         ))
 
         # Set the X and Y limits
-        # The silhouette coefficient can range from -1, 1
-        self.ax.set_xlim([-1, 1])
+        # The silhouette coefficient can range from -1, 1;
+        # but here we scale the plot according to our visualizations
+
+        # l_xlim and u_xlim are lower and upper limits of the x-axis,
+        # set according to our calculated maximum and minimum silhouette score along with necessary padding
```

Comment:
Thank you for not only making the change but for adding in the descriptive comments that explain what is happening here!